tests/php/test-class-amp-setup-wizard-submenu-page.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * Tests for AMP_Setup_Wizard_Submenu_Page class.
+ *
+ * @package AMP
+ */
+
+/**
+ * Tests for AMP_Setup_Wizard_Submenu_Page class.
+ *
+ * @group setup
+ *
+ * @since @todo NEW_ONBOARDING_RELEASE_VERSION
+ *
+ * @covers AMP_Setup_Wizard_Submenu
+ */
+class Test_AMP_Setup_Wizard_Submenu_Page extends WP_UnitTestCase {
+
+	/**
+	 * Setup.
+	 *
+	 * @inheritdoc
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->old_wp_scripts = isset( $GLOBALS['wp_scripts'] ) ? $GLOBALS['wp_scripts'] : null;
+
+		$this->page = new AMP_Setup_Wizard_Submenu_Page( 'amp-options' );
+	}
+
+	/**
+	 * Tear down.
+	 *
+	 * @inheritdoc
+	 */
+	public function tearDown() {
+		parent::tearDown();
+
+		add_action( 'wp_default_scripts', 'wp_default_scripts' );
+
+		if ( function_exists( 'wp_default_packages' ) ) {
+			add_action( 'wp_default_scripts', 'wp_default_packages' );
+		}
+
+		$GLOBALS['wp_scripts'] = new WP_Scripts();
```

Comment:
Why not set it back to `$this->old_wp_scripts`?

```suggestion
		$GLOBALS['wp_scripts'] = $this->old_wp_scripts;
```